doc/rst/dcmp.1.rst

```diff
@@ -42,6 +42,13 @@ OPTIONS
    command. Files walked, started, completed, seconds, files, bytes
    read, byte rate, and file rate.
 
+.. option:: -l, --lite
+
+  lite mode does a comparison of metadata. If metadata is the same,
```

Comment:
Let's say "size and mtime" instead of "metadata" here and in the -h output so the user knows which particular metadata fields we're looking at.

Reply:
@adammoody Got it. The code is updated.